tests/models/test_dagbag.py

```diff
@@ -312,22 +314,20 @@ def process_file(self, filepath, only_if_updated=True, safe_mode=True):
         assert dag_id == dag.dag_id
         assert 2 == dagbag.process_file_calls
 
-    def test_dag_removed_if_serialized_dag_is_removed(self):
+    def test_dag_removed_if_serialized_dag_is_removed(self, dag_maker):
         """
         Test that if a DAG does not exist in serialized_dag table (as the DAG file was removed),
         remove dags from the DagBag
         """
         from airflow.operators.dummy import DummyOperator
 
-        dag = models.DAG(
+        with dag_maker(
             dag_id="test_dag_removed_if_serialized_dag_is_removed",
             schedule_interval=None,
             start_date=tz.datetime(2021, 10, 12),
-        )
-
-        with dag:
+        ) as dag:
             DummyOperator(task_id="task_1")
-
+        dag_maker.create_dagrun()
```

Comment:
We probably don't need/want this on e-- dagbag doesn't care about runs does it?

Reply:
Ok. I will remove it.